includes/utils/class-amp-mutation-utils.php

```diff
@@ -0,0 +1,325 @@
+<?php
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @since 0.7
+ */
+class AMP_Mutation_Utils {
+
+	/**
+	 * The argument if an attribute was removed.
+	 *
+	 * @const array.
```

Comment:
For constants the `@var` tag must be used for the PHPDoc to be valid. This comment applies in multiple constants in this PR.

Reply:
Thanks, this [commit](https://github.com/Automattic/amp-wp/commit/f996673feb2e80585722fc7a9b787e968739f4a9) changes the tags to `@var`.